src/sidebar/components/button.js

```diff
@@ -108,6 +110,11 @@ Button.propTypes = {
   /** callback for button clicks */
   onClick: propTypes.func,
 
+  /** an optional react style object for use on the element--this will override
```

Comment:
This seems pretty standard so I'm in favor of a terse comment. + I could see us using this prop in a few places and It seems best to keep it general.

```
/** Optional inline styling */
style: propTypes.object
```